playbooks/azure/openshift-cluster/launch.yml

```diff
@@ -15,8 +15,8 @@
       url: "{{ item }}"
       dest: "{{ tmp.path }}/"
     with_list:
-    - "http://acs-engine-build-acs-engine-build.svc.ci.openshift.org/acs-engine"
-    - "http://acs-engine-build-acs-engine-build.svc.ci.openshift.org/openshift.json"
+    - "http://acs-engine-build-azure.svc.ci.openshift.org/acs-engine"
```

Comment:
The only thing I'd recommend is we variable-ize this as it once was.  If that's not desirable then please continue with this.